src/main/api/ledger/binance/transaction.ts

```diff
@@ -1,53 +1,89 @@
+import LedgerApp from '@binance-chain/javascript-sdk/lib/ledger/ledger-app'
+import Transport from '@ledgerhq/hw-transport'
+import { Client, getDerivePath, getPrefix } from '@xchainjs/xchain-binance'
+import { Address, TxHash } from '@xchainjs/xchain-client'
+import { Asset, AssetBNB, BaseAmount, baseToAsset } from '@xchainjs/xchain-util'
 import * as E from 'fp-ts/lib/Either'
 
-import { LedgerErrorId } from '../../../../shared/api/types'
+import { LedgerError, LedgerErrorId, Network } from '../../../../shared/api/types'
+import { toClientNetwork } from '../../../../shared/utils/client'
+import { isError } from '../../../../shared/utils/guard'
 
-export const send = () =>
-  Promise.reject(
-    E.left({
+/**
+ * Sends `MsgSend` message using Ledger
+ */
+export const send = async ({
+  transport,
+  network,
+  sender,
+  recipient,
+  amount,
+  asset,
+  memo
+}: {
+  transport: Transport
+  amount: BaseAmount
+  network: Network
+  sender?: Address
+  recipient: Address
+  asset?: Asset
+  memo?: string
+}): Promise<E.Either<LedgerError, TxHash>> => {
+  try {
+    const clientNetwork = toClientNetwork(network)
+    const prefix = getPrefix(clientNetwork)
+    const derivePath = getDerivePath(0)
+
+    const app = new LedgerApp(transport)
+    const client = new Client({ network: clientNetwork })
+
+    await app.showAddress(prefix, derivePath)
```

Comment:
Do we really need this call of `showAddress`^? By calling `useLedgerSigningDelegate` Binance app on Ledger is showing address + txs address as well